spec/requests/atmosphere/api/actions_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe Atmosphere::Api::V1::ActionsController do
+  include ApiHelpers
+
+  describe 'GET /actions' do
+
+    context 'when unauthenticated' do
+      it 'returns 401 Unauthorized error' do
+        get api("/actions")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.